docs/source/pillars/information_governance.md

```diff
@@ -86,6 +86,12 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You report on and share outcomes of each audit of your TRE organisation with the required bodies
   - This may included regulatory bodies or the organisations that manage accreditations you have
   - Mandatory
+* - You log user actions, network connection attempts, infrastructure actions and data access within a TRE
+  - The greater the number of (and diversity of) actions logged, the higher the chance that any bad behaviours, intentional or unintentional, are spotted.
+    Bad behaviours in this context are anything that could compromise the security of the SRE, or result in the accidental discolure of data outside the TRE.
+    Any measures that are valuable for reporting should be collected, for example, the number of times that unauthorised access to the TRE was blocked.
+    Where possible, this reporting should be made publicly available, to incentivise good governance and operation, and should be automated, to enable regular compliance checks.
```

Comment:
I'm not a fan of saying it's "mandatory" to do something "where possible". Could we move this to a separate optional statement?